packages/frontend/src/features/sqlRunner/transformers/useSqlChart.ts

```diff
@@ -24,23 +25,23 @@ export const useSqlChart = (
             }),
         [rows, columns],
     );
-
-    const visTransformer = useMemo(() => {
-        if (isCartesianChartSQLConfig(config)) {
+    return useAsync(async () => {
```

Comment:
refactor to handle `getEchartsSpec` having different signatures across pie vs. cartesian